P202.Training.WCF/Handlers/UpdateUserHandler.cs

```diff
@@ -18,7 +18,7 @@ public UpdateUserHandler(IUsersService usersService)
         public override Response Handle(UpdateUserRequest request)
         {
             var response = CreateTypedResponse();
-            response.User = _usersService.UpdateUser(request.User);
+            _usersService.UpdateUser(request.User);
```

Comment:
If we are not going to set the User property anymore please remove it from the `UpdateUserResponse` class

Reply:
I'd update the Response and return the updated user. Already discussed this with Luis to do so.

Reply:
Whatever you decide it's fine by me as long as we keep it consistent along the project. Should we vote @DanielAltamirano @darkainsoul @alcabelu @gmsoft ?

Reply:
I'm ok keeping the response

Reply:
i think is good response the user update 

but i do not if have code response 200, 201, 204